hub/auto/unstructured/image_classification.py

```diff
@@ -40,6 +40,26 @@ def _set_name_from_path(path: Path) -> str:
     return path.parts[-3]
 
 
+def ingestion_summary(src: str, skipped_files: list):
+    columns, lines = os.get_terminal_size()
+
+    mid = int(columns / 2)
+    for i in range(columns - 20):
+        print("=", end="")
+        if i == mid - 10:
+            print(" Ingestion Summary ", end="")
+    print("\n")
+
+    for root, dirs, files in os.walk(src):
+        level = root.replace(src, "").count(os.sep)
+        indent = " " * 6 * (level)
+        print("{}{}/".format(indent, os.path.basename(root)))
+        subindent = " " * 6 * (level + 1)
+        for f in files:
+            if f in skipped_files:
+                print("{}{} -- Skipped".format(subindent, f))
+
+
 class ImageClassification(UnstructuredDataset):
     def __init__(self, source: str):
         """Convert an unstructured dataset to a structured dataset.
```

Comment:
the progress bar's description is the full path to the dataset, but in the screen shot you provided is extremely long. can we just use the last folder?